packages/front-end/components/HomePage/NorthStar.tsx

```diff
@@ -12,7 +12,8 @@ import { ExperimentInterfaceStringDates } from "back-end/types/experiment";
 
 const NorthStar: FC<{
   experiments: ExperimentInterfaceStringDates[];
-}> = ({ experiments }) => {
+  project: string;
+}> = ({ experiments, project }) => {
```

Comment:
You can avoid all these new props.  The currently selected project is available via context anywhere in the app.

```ts
const {project} = useDefinitions()
```